cmd/minikube/cmd/start.go

```diff
@@ -638,6 +639,29 @@ func hostDriver(existing *config.ClusterConfig) string {
 	return h.Driver.DriverName()
 }
 
+// validateProfileName makes sure that new profile name not duplicated with any of machine names in existing multi-node clusters.
+func validateProfileName(existing *config.ClusterConfig) {
+	if existing != nil {
```

Comment:
Shouldn't this be

```go
if existing == nil {
  return
}
```

Either way, it doesn't look like `existing` is used at all in this function. If that's intentional, I would probably remove this if statement and rewrite the function header to:


```go
func validateProfileName(_ *config.ClusterConfig) {
```

Reply:
Remove argument, and changed to call function only existing is not nil.

```
if existing != nil {
		upgradeExistingConfig(existing)
		validateProfileName()
	}
```